src/parsec_client/operations/psa_export_key.md

```diff
@@ -27,25 +27,27 @@ that is equivalent to key.
 
 For standard key types, the output format is as follows:
 
-- For symmetric keys, including MAC keys, the format is the raw bytes of the key.
+- For symmetric keys, including HMAC keys, the format is the raw bytes of the key.
```

Comment:
Note that the [spec differentiates](https://armmbed.github.io/mbed-crypto/html/api/keys/management.html#psa_export_key) between "HMAC keys that are longer than the underlying hash algorithm block size" and those that are equal or tinier.
Here that would make the choice that the format would always be the raw bytes of the key.